### ISSUE TITLE
Listbox scroll focus not accounting for padding

### ISSUE THREAD

**Issue by Confidenceman02** [Confidenceman02/elm-select#6]
When there are overflowed menu options you can visually focus them by using arrow keys to essentially scroll through the list.
When keyboard focusing a menu option that is not visible the listbox auto scrolls to allow for the menu option to be visible. 
The `<ul>` scene does not account for the padding on the top and bottom and so a small part of the menu option is not visible.

## Proposed solution
Because we are currently querying the `<ul>` scene, we will need to adjust the viewport scroll to account for padding that may be partially obstructing the menu option. 

Ideally we can set the top and bottom padding as variables to future proof any changes to them.
